mclocations/minecraft.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+from mcrcon import MCRcon
+
+from mclocations.models import Player, Position
+
+
+class MinecraftServer:
+    def __init__(self, host, password, port=25575):
+        self.mcr = MCRcon(host, password, port)
+        self.mcr.connect()
+
+    def disconnect(self):
+        self.mcr.disconnect()
+
+    def get_players(self):
+        players = []
+        player_names = MinecraftServer._parse_list(self.mcr.command('/list'))
+
+        for player_name in player_names:
+            position = MinecraftServer._parse_position(self.mcr.command(f'/data get entity {player_name} Pos'))
+            dimension = MinecraftServer._parse_dimension(self.mcr.command(f'/data get entity {player_name} Dimension'))
+            players.append(Player(player_name, position, dimension))
+
+        return players
+
+    @classmethod

```

Comment:
I guess I have more reading to do. IDK what this annotation means.

Reply:
It's a `static` method from Java. You just need the class and not an instance to use this method.